modules/oadp-1-4-1-release-notes.adoc

```diff
@@ -0,0 +1,138 @@
+With this update, the restore CR status for a restore will correctly proceed to `Completed` or `Failed` status.
+link:https://issues.redhat.com/browse/OADP-3227[OADP-3227]
+
+.Restoring BuildConfig Build from a different cluster is successful without any errors
+
+Previously, when performing a restore of the `BuildConfig` Build resource from a different cluster, the application generated an error on TLS verification to the internal image registry. The resulting error was `failed to verify certificate: x509: certificate signed by unknown authority` error.
+With this update, the restore of the `BuildConfig` build resources to a different cluster can proceed successfully without generating the `failed to verify certificate` error.
+link:https://issues.redhat.com/browse/OADP-4692[OADP-4692]
+
+.Restoring an empty PVC is successful
+
+Previously, downloading data failed while restoring an empty persistent volume claim (PVC). It failed with the following error:
+----
+data path restore failed: Failed to run kopia restore: Unable to load
+    snapshot : snapshot not found
+----
+With this update, the downloading of data proceeds to correct conclusion when restoring an empty PVC and the error message is not generated. 
+link:https://issues.redhat.com/browse/OADP-3106[OADP-3106]
+
+.There is no Velero memory leak in CSI and DataMover plugins
+
+Previously, a Velero memory leak was caused by using the CSI and DataMover plugins. When the backup ended, the Velero plugin instance was not deleted and the memory leak consumed memory until an `Out of Memory` (OOM) condition was generated in the Velero pod. With this update, there is no resulting Velero memory leak when using the CSI and DataMover plugins.
+link:https://issues.redhat.com/browse/OADP-4448[OADP-4448]
+
+.Post-hook operation does not start before the related PVs are released
+
+Previously, due to the asynchronous nature of the Data Mover operation, a post-hook might be attempted before the related pods' persistent volumes (PVs) were released by the Data Mover persistent volume claim (PVC). This problem would cause the backup to fail with a `PartiallyFailed` status.
+With this update, the post-hook operation is not started until the related PVs are released by the Data Mover PVC, eliminating the `PartiallyFailed` backup status.
+link:https://issues.redhat.com/browse/OADP-3140[OADP-3140]
+
+.Deploying a DPA works as expected in namespaces with more than 37 characters
+
+While installing the OADP Operator in a namespace with more than 37 characters to create a new DPA, labeling the "cloud-credentials" Secret will fail and the DPA reports the following error:
+----
+The generated label name is too long.
+----
+With this update, deploying a DPA does not fail in namespaces with more than 37 characters in the name.
```

Comment:
```suggestion
With this update, creating a DPA does not fail in namespaces with more than 37 characters in the name.
```

Reply:
use consistent language. You use create before, so use it again here instead of deploy